CHANGELOG.md

```diff
@@ -0,0 +1,54 @@
+# Changelog DeviceSwap
+
+## Table of Contents
+
+- [r1.1](#r11)
+
+**Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until it has been released. For example, changes may be reverted before a release is published. For the best results, use the latest published release.**
+
+# r1.1
+## Release Notes
+
+This release contains the definition and documentation of
+* device-swap v0.1.0-rc.1
+
+The API definition(s) are based on
+* Commonalities v0.4.0
+* Identity and Consent Management v0.2.0
+
+## device-swap v0.1.0-rc.1
+
+**device-swap v0.1.0-rc.1 is the 1st release candidate of the version 0.1.0**
+
+- 0.1.0-rc.1 Device Swap API definition **with inline documentation**:
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/DeviceSwap/blob/r1.1/code/API_definitions/device-swap.yaml)
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/DeviceSwap/blob/r1.1/code/API_definitions/device-swap.yaml&nocors)
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/DeviceSwap/blob/r1.1/code/API_definitions/device-swap.yaml)
+    
+## What's Changed
+### Added
+* Update CODEOWNERS by @ShutingQing in https://github.com/camaraproject/DeviceSwap/pull/2
+* Update README.md w/ Meeting info by @wrathwolf in https://github.com/camaraproject/DeviceSwap/pull/7
+* Update CODEOWNERS by @bigludo7 in https://github.com/camaraproject/DeviceSwap/pull/8
+* Update MAINTAINERS.MD by @bigludo7 in https://github.com/camaraproject/DeviceSwap/pull/9
+* Update CODEOWNERS - adding Jorge Garcia Hospital by @bigludo7 in https://github.com/camaraproject/DeviceSwap/pull/12
```

Comment:
I would omit these kind of "changes" from the CHANGELOG, they are internal management changes and have no value for the audience (they still documented in the "full" changelog).

Reply:
solved